spec/controllers/users_controller_spec.rb

```diff
@@ -28,4 +28,17 @@
       expect(user.reload.name).to eq(user.name)
     end
   end
+
+  describe 'DELETE users#destroy' do
+    it 'deletes the user' do
+      user = create(:user)
+      expect(response).to have_http_status(:ok)
+      expect { delete :destroy, params: { id: user.id } }.to change(User, :count).by(-1)
+    end
+
+    it 'returns an error if the user id is invalid' do
+      delete :destroy, params: { id: 'invalid-id' }
+      expect(response).to have_http_status(:not_found)
```

Comment:
This still doesn't actually check that something was deleted - just checks that a :not_found was returned.

We need to actually check that a record in the database was deleted

Reply:
The test at line 33 does that.
Not sure test at line 39 is any good - please let me know.

Reply:
Sorry meant to say **not** deleted. 

This should do exactly as 36, but instead check that it wasn't changed